### PR TITLE
Set correct edge properties for analytic applications

### PR DESCRIPTION
**Pull request by nicelhc13**

### CODE REVIEW

File: scripts/bench_python_cpp_algos.py [hunk 1]
```diff
@@ -292,10 +292,10 @@ def run_all_gap(args):
         },
     ]
 
-    def load_graph(graph_path):
+    def load_graph(graph_path, edge_properties=None):
```
Comment (arthurp) [inline scripts/bench_python_cpp_algos.py:295]: Do you want to limit `node_properties` as well? This should be done throughout if you want to.

File: scripts/bench_python_cpp_algos.py [hunk 2]
```diff
@@ -292,10 +292,10 @@ def run_all_gap(args):
         },
     ]
 
-    def load_graph(graph_path):
+    def load_graph(graph_path, edge_properties=None, node_properties=None):
         print(f"Running {args.application} on graph: {graph_path}")
         with time_block("read propertyGraph"):
-            graph = PropertyGraph(graph_path)
+            graph = PropertyGraph(graph_path, edge_properties=edge_properties, node_properties=node_properties)
```
Comment (ddn0) [inline scripts/bench_python_cpp_algos.py]: as I understand the benchmarks, none of them actually use the node properties from the inputs, so we can just set `node_properties=[]` always.

**Review by ddn0 — APPROVED** [review]

File: scripts/bench_python_cpp_algos.py (hunk 2 above)
Comment (arthurp) [reply scripts/bench_python_cpp_algos.py]: If this is true. I am with Donald.

**Review by arthurp — APPROVED** [review]

File: scripts/bench_python_cpp_algos.py (hunk 2 above)
Comment (nicelhc13) [reply scripts/bench_python_cpp_algos.py]: Yes you are right

**Pull request merged**